lib/src/client.dart

```diff
@@ -1586,9 +1586,9 @@ class Client extends MatrixApi {
       );
 
       /// Timeout of 0, so that we don't see a spinner for 30 seconds.
-      final syncFuture = _sync(timeout: Duration.zero);
+      firstSyncReceived = _sync(timeout: Duration.zero);
       if (waitForFirstSync) {
```

Comment:
Are you sure this would work for the first sync? 

The docs say: The server need only populate this property for an incremental /sync (i.e., one where the since parameter was specified) https://spec.matrix.org/v1.9/client-server-api/#e2e-extensions-to-sync

We could just use /keys/query here

Reply:
I'm not sure that I get this. It works the same way as before but now you can await it from outside as well

Reply:
Hm not sure either, I think I confused getting the device list with /sync device_list updates